README.md

```diff
@@ -39,7 +39,7 @@ that can be explored in the browser.
 
 ### Building on Macbook M1
 
-Building Reveal on Macbook M1 migth require some special care.
+Building Reveal on Macbook M1 might require some special care.
```

Comment:
Interestingly I didn't have any issues setting this up on my M2 MacBook. Probably make sense to keep this around somewhat longer, but I think this entire section can be deleted at some point.